README.txt

```diff
@@ -3,4 +3,5 @@ This modules allows you to use the php-github-api library with a Drupal site.
 
 This module depends on composer_manager.
 
-Note: The github password is stored in plain text in the database.
+Note: The github password is stored in plain text in the database. You may avoid
+this by directly storing a personal access token in the github_api_token variable.
```

Comment:
The documentation is now out of date. We now use the username and password to fetch a personal token.